resources/[gameplay]/gcshop/horns/horns_c.lua

```diff
@@ -385,8 +385,13 @@ hornsTable = {
 	[382] = "Gibberish",
 	[383] = "Haka",
 	[384] = "Turn Down For What",
-}
-
+}       [385] = "Kylie",
```

Comment:
Keep in mind for following PR's to close the table at the end of the list, not halfway. If this isn't corrected the gcshop will fail to load and give an error.

Reply:
Did u fIX IT ?

Reply:
GG